### PR TITLE
Add ReStable@0.1.2 w/ git auto-update config

### LINKED ISSUE #9590: [Request] Add ReStable
**Library name:** ReStable
**Git repository url:** https://github.com/micc83/ReStable

### PR DESCRIPTION
**Pull request by x09326**
Pull request for issue: #9590 
Related issue(s): # #
@kennynaoh Please help me review this pull requset, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/micc83/ReStable
 * Official website (optional, not the repository):https://codeb.it/restable
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:171
   - Count of watchers:6
   - Count of forks:44
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9590, cc @micc83

### CODE REVIEW

**Comment by micc83** [comment]
I saw you cc'd me. Is there something I should do to proceed?

**Comment by PeterDaveHello** [comment]
@micc83 Just let you know that we're going to host this library :) You can help us review the commit(s) in this PR if you would like :D

**Comment by kennynaoh** [comment]
LGTM

File: ajax/libs/ReStable/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "keywords": [
+    "lightweight",
+    "jquery",
+    "plugin",
+    "table"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mic83/ReStable.git",
```
Comment (PeterDaveHello) [inline ajax/libs/ReStable/package.json]: 404 not found ...

Comment (x09326) [inline ajax/libs/ReStable/package.json]: @kennynaoh 
I have modified it, please help me review it, thanks.

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@x09326 thanks for this PR, maybe you can consider to solve the high priority issues and issues not from our team first, thank you.

**Pull request merged**